  string name
  float64[] center1
  float64[] quaternion1
  float64[] dimensions1
  float64[] center2
  float64[] quaternion2
  float64[] dimensions2